tests/LocalBearTestHelper.py

```diff
@@ -14,6 +14,13 @@
 
 @contextmanager
 def execute_bear(bear, *args, **kwargs):
+    """
+    Execute the bear routine.
```

Comment:
We usually leave a newline between the function doc and the param descriptions. I'm not sure if it's for some parser, but I'd suggest the same here.